### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: fuzz/fuzz.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include "SDL_image.h" //fuzzer target
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this to indicate what's being fuzzed?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: You don't need to do a git clone - `ADD .` is sufficient. Can we add a package stage here?

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sdlimage-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Tests has been renamed to testsuite. You may omit the image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Is it necessary to commit such a large corpus? Usually 1-2 files are sufficient. Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I wanted to include one file from every format type supported by the library to help detect possible crashes faster, as I imagine it would take a long time for the fuzzer engine to mutate an AVIF to a valid JXL format, for example. I can remove some if you'd like, I just want your input first. 

All other changes have been made!

**Review by xansec — APPROVED** [review]
As long as the corpus isn't too large (takes up too much space) I think we're ok here. Larger corpora should be handled by git-lfs, and probably doesn't even belong on git in the first place. Something to keep in mind for future work. LGTM!

**Comment by xansec** [comment]
@capuanob I had to merge the `CMakeLists.txt` manually due to upstream changes, and it doesn't seem to be taking... Can you double check that the `-fsanitize` flags are all in the right place now? 
https://github.com/mayhemheroes/SDL_image/runs/7614823881?check_suite_focus=true

**Pull request merged**